web-2.0/src/main/java/org/openmrs/module/allergyui/page/controller/AllergyPageController.java

```diff
@@ -240,7 +240,7 @@ private AllergyReaction getAllergyReactionByConcept(Allergy allergy, Concept con
 		//Unknown should come first
 		if (concept != null) {
 			for (Concept c : concept.getSetMembers()) {
-				if (Allergen.OTHER_NON_CODED_UUID.equals(c.getUuid())) {
+				if (Allergen.getOtherNonCodedConceptUuid().equals(c.getUuid())) {
```

Comment:
@jdegraft can you add this method to the allergyapi module?
The reason being that some are running openmrs platform versions before 2.0 and hence rely on the allergyapi module which does not yet have the new Allergen.getOtherNonCodedConceptUuid() method. Does this make sense? :)